packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx

```diff
@@ -67,6 +69,13 @@ const CreateDaoCollateral: React.FC<{
     formState: { errors },
   } = methods;
 
+  useEffect(() => {
+    if (isExistingToken) {
+      setValue('isScheduleNewDaoToken', 0);
+      setValue('isChallengeNewDaoToken', 0);
+    }
+  }, [isExistingToken, setValue]);
```

Comment:
Can you explain why this useEffect is needed as it looks to me the form field's onChange() handler would update the values.

Reply:
this use effect trigger once when component is mounted and if previouse basicInfo.isExistingToken is true.
this is needed, in the senarios of for example: isScheduleNewDaoToken set to true but user go back to basic info and change to using existing token, so when user reach this screen these fariables are set to default again.